src/features/Movie/People/styled.js

```diff
@@ -7,6 +7,11 @@ export const Wrapper = styled.div`
 	}
 `
 
+export const PersonPosterWrapper = styled.div`
+	height: 178px;
+    background: #C4C4C4;
+`
+
 export const StyledActorTile = styled.article`
 	padding: 8px;
 	background-color: #fff;
```

Comment:
Maybe colors from theme.js file can be used here? ;)

Reply:
Yes, my bad. I am correcting